src/middleware.js

```diff
@@ -90,6 +100,7 @@ function _validateParams(requestOptions) {
 
 module.exports = {
     init,
+    initAsync,
```

Comment:
New method needs to be documented in readme

Reply:
added function description to readme file